packages/core/pluggableElementTypes/renderers/ServerSideRendererType.ts

```diff
@@ -125,17 +157,17 @@ export default class ServerSideRenderer extends RendererType {
     )
     // const result = await renderRegionWithWorker(session, serializedArgs)
 
-    this.deserializeResultsInClient(result, args)
-    return result
+    const deserialized = this.deserializeResultsInClient(result, args)
+    return deserialized
   }
 
-  getExpandedGlyphRegion(region: IRegion, renderArgs: RenderArgs) {
+  getExpandedGlyphRegion(region: IRegion, renderArgs: RenderArgsDeserialized) {
     if (!region) return region
     const { bpPerPx, config } = renderArgs
-    const maxFeatureGlyphExpansion = readConfObject(
-      config,
-      'maxFeatureGlyphExpansion',
-    )
+    const maxFeatureGlyphExpansion =
```

Comment:
any reason for this extra guard for config being undefined?

Reply:
because readConfObject now throws if it is not passed a conf object